client/lib/post-normalizer/utils.js

```diff
@@ -1,3 +1,5 @@
+/** @ssr-ready **/
```

Comment:
Yeah, not `@ssr-ready`able because this module depends on the DOM, and doesn't have proper fallbacks.